OYarotski/02.Tables,links/1.Calendar.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html> 
+<HTML lang="ru"> 
+    <HEAD> 
+      <META charset="utf-8"> 
+      <TITLE>Таблицы</TITLE> 
+    </HEAD> 
+    <BODY> 
+        <table border=1 width="400" align="center">
+        <caption>Месяц моего дня рождения</caption>
+        <col width="40"><col width="30"><col width="30"><col width="30"><col width="30"><col width="30"><col width="30"> 
+        <tr><th colspan="6">Май</th><tr></tr>
+        <tr><th>ПН</td><td></td><td>6</td><td>13</td><td>20</td><td>27</td><tr>
```

Comment:
the is open tag tr in the end of the lines 12-18. should be close part, not open part